src/rgw/rgw_main.cc

```diff
@@ -301,6 +304,7 @@ int radosgw_Main(int argc, const char **argv)
   rgw_init_resolver();
   rgw::curl::setup_curl(fe_map);
   rgw_http_client_init(g_ceph_context);
+  rgw_kmip_client_init(*new RGWKMIPManagerImpl(g_ceph_context));
```

Comment:
can init() just take `g_ceph_context` to hide the dependency on `RGWKMIPManagerImpl`?

Reply:
should we be relying on g_ceph_context?